### DISCUSSION TITLE
Compatibility with affiliate Plugins

### DISCUSSION THREAD

**Discussion by itsinalis** [woocommerce/woocommerce#45083] [WooCommerce, Blocks, and Site Editing]
Hello Guys!

I need to set an affiliate management system for my webshop.

We use Woocommerce Blocks for cart and checkout pages, and it’s really good compared to the default Woocommerce cart and checkout.

Although, none of the Affiliate plugins that I’ve tried so far is compatible with Woocommerce blocks, which causes me a huge problem.

Please, this is urgent!

We need your help!!

**Comment by nerrad** [comment]
Hi there @itsinalis  👋 

Thanks for taking the time to make us aware that affiliate plugins that you've tried aren't working with the Checkout and Cart blocks. These blocks are still in early access (via the feature plugin) and it's expected that some existing Woo Extensions that integrate with the current checkout flow in WooCommerce core will not work with the blocks. 

We're currently working on adding extensibility for the blocks so extensions have a way to make sure they integrate with the new flows we're developing. Once that is done we'll have docs prepared and will be reaching out to Woo Extension developers to help them with getting their extensions integrated.

Until then you can keep on top of the [status of the Cart & Checkout Blocks in this documentation](https://docs.woocommerce.com/document/cart-checkout-blocks-support-status/)

I'm going to close this issue since we only keep open issues that are actionable in some way. I've taken note of your request and we'll make sure to ensure that affiliate extensions will be able to integrate with the new flows.

**Comment by itsinalis** [comment]
Hi there,

Thank you for your email!

Have a good week!

Best,
Jacob
